### ISSUE TITLE
Use Vagrant to create and publish the VM

### ISSUE THREAD

**Issue by gilesknap** [epics-training/training-vm#21]
As discussed at the EPICS collaboration meeting.

**Comment by gilesknap** [comment]
Came here to report that I have been playing with vagrant for 1 hour and already have a rocky base vm installing completing the ansible steps in this repo. Will make a PR soon.

**Comment by ralphlange** [comment]
Nice! I'm getting stuck at the gnome-GUI not being started.

**Comment by gilesknap** [comment]
I think I have this working. But have to go play Halo with my son right now. Will verify and PR later today.
One question @ralphlange - is there a trick to getting the EPICS binaries added to epics-dev's PATH?

**Comment by ralphlange** [comment]
Priorities!
Thank you so much.

The magic:
Controlled by a [variable setting](https://github.com/ralphlange/training-vm/blob/c163269c82d7e6f1987f1e5844ef16c35d513beb/ansible/roles/epics-modules/vars/main.yml#L9), the module build script [adds a fragment](https://github.com/ralphlange/training-vm/blob/c163269c82d7e6f1987f1e5844ef16c35d513beb/ansible/roles/epics-modules/tasks/build_module.yml#L100) that gets [added to the path](https://github.com/ralphlange/training-vm/blob/c163269c82d7e6f1987f1e5844ef16c35d513beb/ansible/roles/epics-modules/tasks/build_module.yml#L107) further down.

**Comment by gilesknap** [comment]
So I have this working fully but needed to switch distro to achieve that. Fedora does not exhibit any of the issues Rocky has and I did not need to do anything to fix them except switch to the latest official fedora box. (except work out how to make the root filesystem big enough to accommodate gnome and java)

I went down many rabbit holes looking for a Rocky solution and it's a shame because apart from that it has been a really smooth process to learn vagrant and get the ansible playbook deployed into a VM.
